framework/test/driver/BundleResourceTest.cpp

```diff
@@ -161,6 +161,9 @@ void testInvalidResource(const Bundle& bundle)
 
   US_TEST_CONDITION(res.GetChildren().empty(), "Check empty children")
   US_TEST_CONDITION(res.GetSize() == 0, "Check zero size")
+  US_TEST_CONDITION(res.GetCompressedSize() == 0, "Check zero compressed size")
+  US_TEST_CONDITION(res.GetLastModified() == 0, "Check zero last modification time")
+  US_TEST_CONDITION(res.GetCrc32() == 0, "Check zero CRC-32")
```

Comment:
It'd be great if we have a positive test for non-zero CRC-32. Maybe a good test point to add that is in `testCompressedResource`. Something like

```cpp
 US_TEST_CONDITION(res.GetCrc32() == XXXX, "Check valid CRC-32")
```
Apart from that I have no flags from this submission. Thanks!

Reply:
Hi Krish, I didn't add any positive tests, because it looked like everything else used the helper 'checkResourceInfo', I didn't want to have to force *all* the positive tests to now need to provide the CRC-32, and I didn't have a sense for which specific test should have a CRC-32 check.